src/super_gradients/recipes/dataset_params/cityscapes_stdc_seg50_dataset_params.yaml

```diff
@@ -5,28 +5,28 @@ defaults:
 train_dataset_params:
   transforms:
     # for more options see common.factories.transforms_factory.py
-    - ColorJitterSeg:
+    - SegColorJitter:
```

Comment:
All the detection Names and classes start with "Detection", while our segmentation names finish with Seg.
Since I had to rename all the segmentation transforms, I thought that it would be better to follow the same convention as with Detection. And because we want the names and transforms to match, I also changed the names